Makefile

```diff
@@ -17,13 +17,19 @@ dev:
 	@echo "-> Configure the development envt."
 	./configure --dev
 
```

Comment:
@ziadhany please add a command here to provision db, I am getting this error `django.db.utils.OperationalError: connection failed: FATAL:  database "purl-sync" does not exist`

Reply:
Done